### ISSUE TITLE
README should include instructions for manually adding KeyCastr to the Security & Privacy Accessibility list

### ISSUE THREAD

**Issue by akitchen** [keycastr/keycastr#91]
The instructions in the app might not be clear enough. And in current version(s) of OS X, brew cask seems not to be able to automate this for us.

**Comment by akitchen** [comment]
```
○ → brew cask install keycastr
==> Satisfying dependencies
==> Downloading https://github.com/keycastr/keycastr/releases/download/v0.9.4/KeyCastr.app.zip
Already downloaded: /Users/akitchen/Library/Caches/Homebrew/Cask/keycastr--0.9.4.zip
==> Verifying checksum for Cask keycastr
==> Installing Cask keycastr
==> Moving App 'KeyCastr.app' to '/Applications/KeyCastr.app'.
==> Enabling accessibility access
Warning: Accessibility access cannot be enabled automatically on this version of macOS.
See System Preferences to enable it manually.
🍺  keycastr was successfully installed!
```

**Comment by akitchen** [comment]
<img width="673" alt="screen shot 2017-07-17 at 10 09 17" src="https://user-images.githubusercontent.com/378648/28259736-ffad8ff6-6ad7-11e7-8090-be0bc760f52a.png">

**Comment by tedyoung** [comment]
How do I add KeyCastr.app to the Security & Privacy -> Accessibility? You've shown the "after", but not the "before". Some more detailed instructions here would help.

**Comment by tedyoung** [comment]
Figured it out -- had unlock (bottom-left), then click on the +, then find KeyCastr.app in Applications, and then "Open" it and it's added to the list.
